### ISSUE TITLE
Search API doesn't find package

### ISSUE THREAD

**Issue by ermshiperete** [NuGet/NuGetGallery#7496]
### Describe the bug
I try to use the query API to get information about the `SIL.Windows.Forms` nuget package. This works for other packages, but `SIL.Windows.Forms` returns nothing.

### To Reproduce

    curl 'https://api-v2v3search-0.nuget.org/query?q=packageid:SIL.Windows.Forms'

### Expected behavior

Output similar to the output that I get when I run (but of course for `SIL.Windows.Forms`, not `SIL.Windows.Forms.WritingSystems`) the following command:

```
curl 'https://api-v2v3search-0.nuget.org/query?q=packageid:SIL.Windows.Forms.WritingSystems'

{
  "@context": {
    "@vocab": "http://schema.nuget.org/schema#",
    "@base": "https://api.nuget.org/v3/registration3/"
  },
  "totalHits": 1,
  "lastReopen": "2019-08-29T17:22:58.0949519Z",
  "index": "v3-lucene2-v2v3-20171018",
  "data": [
    {
      "@id": "https://api.nuget.org/v3/registration3/sil.windows.forms.writingsystems/index.json",
      "@type": "Package",
      "registration": "https://api.nuget.org/v3/registration3/sil.windows.forms.writingsystems/index.json",
      "id": "SIL.Windows.Forms.WritingSystems",
      "version": "7.0.0",
      "description": "SIL.Windows.Forms.WritingSystems contains Windows Forms UI elements for displaying and editing writing systems as defined by the SIL.WritingSystems assembly.",
      "summary": "",
      "title": "SIL.Windows.Forms.WritingSystems",
      "licenseUrl": "https://www.nuget.org/packages/SIL.Windows.Forms.WritingSystems/7.0.0/license",
      "projectUrl": "https://github.com/sillsdev/libpalaso",
      "tags": [],
      "authors": ["SIL International"],
      "totalDownloads": 1494,
      "verified": false,
      "versions": [
        {
          "version": "7.0.0",
          "downloads": 0,
          "@id": "https://api.nuget.org/v3/registration3/sil.windows.forms.writingsystems/7.0.0.json"
        }
      ]
    }
  ]
}
```

### Actual output

Instead of the expected output I get:

```
curl 'https://api-v2v3search-0.nuget.org/query?q=packageid:SIL.Windows.Forms'

{
  "@context": {
    "@vocab": "http://schema.nuget.org/schema#",
    "@base": "https://api.nuget.org/v3/registration3/"
  },
  "totalHits": 0,
  "lastReopen": "2019-08-29T17:27:55.1699468Z",
  "index": "v3-lucene2-v2v3-20171018",
  "data": []
}
```

**Comment by loic-sharma** [comment]
Hello. This is confusing but expected behavior. You will need to update your search query to include SemVer v2.0.0 results: https://api-v2v3search-0.nuget.org/query?q=packageid:sil.windows.forms&prerelease=true&semVerLevel=2.0.0

The search service by default excludes [results that are SemVer v2.0.0](https://github.com/NuGet/Home/wiki/SemVer2-support-for-nuget.org-%28server-side%29#identifying-semver-v200-packages). The `SIL.Windows.Forms` is a SemVer v2.0.0 package because it depends on the SemVer v2.0.0 package [TagLibSharp-patched v2.2.0-beta.1](https://www.nuget.org/packages/TagLibSharp-patched/2.2.0-beta.1). For more information, please refer to the Search API documentation: https://docs.microsoft.com/en-us/nuget/api/search-query-service-resource

As a side note, we've updated the search endpoint to: `https://azuresearch-usnc.nuget.org/query`. Please update your URLs as we will delete the legacy `api-v2v3search-x.nuget.org` endpoints in the future.

**Comment by ermshiperete** [comment]
Thanks! I suspected that I was missing something...
